Sources/RayTracer.Library/Shapes/Disc.cs

```diff
@@ -0,0 +1,40 @@
+﻿using RayTracer.Library.Mathematics;
+
+namespace RayTracer.Library.Shapes;
+
+public class Disc : IIntersectable
+{
+    public Vector3 Center;

```

Comment:
Please expose these as get-only props
Normal can be private because there is a method for it